test/integration/basic_test.go

```diff
@@ -106,14 +106,22 @@ func compareExpectedWithActualContains(expected string, actual string) error {
 }
 
 func compareExpectedWithActualEquals(expected string, actual string) error {
-
 	if actual != expected {
 		return fmt.Errorf("Output did not match. Expected: %s, Actual: %s", expected, actual)
 	}
 
 	return nil
 }
 
+func findExpectedMatchInActual(expectedMatch, actual string) error {
+	r, _ := regexp.Compile(expectedMatch)
+	if !r.MatchString(actual) {
+		return fmt.Errorf("Actual output %s doesn't contain match %s", actual, expectedMatch)
```

Comment:
@gbraad , @hferentschik , `match` is totally different from `contain`.

`match` do regex match whereas `contain` check if it is part of string or not.
`contain` doesn't able to perform regex match.

https://play.golang.org/p/g2dY9piGQK

If we are changing the implementation of `compareExpectedWithActualContains` then IMO we are creating confusion by having two behavior handle by it.